docker/build/rpi/install_deps.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+apt-get -qq update
+
+apt-get -qq install --no-install-recommends -y \
```

Comment:
Everything here has already been installed into the `deps` (and thus `frigate`) image upon which this rpi image is based

Reply:
this isn't true in my experience, I removed these but it failed saying it could not find the gpg command

Reply:
but to be fair I have made a lot of changes so that might be part of it too

Reply:
I will push up all my changes later today